<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBar</class>
 <widget class="QWidget" name="ToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>56</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton {
   color: #333;
   background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, 
          stop: 0 #fff,
          stop: 0.5 #eee, 
          stop: 1 #999);
   border-width: 2px;
   border-color: #333;
   border-style: solid;
   border-radius: 5;
   padding: 0px;
   font-size: 24px;
   min-width: 34px;
   max-width: 34px;
   min-height: 34px;
   max-height: 34px;
}

QToolButton:disabled {
  background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, 
          stop: 0 #fff,
          stop: 0.5 #eee, 
          stop: 1 #bbb);
   border-color: #aaa;
   color: #aaa;
}

QToolButton:pressed {
   background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, 
          stop: 0 #999,
          stop: 0.5 #eee, 
          stop: 1 #fff);
}


QToolButton:checked {
   border-color: #b00;
   background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, 
          stop: 0 #999,
          stop: 0.5 #eee, 
          stop: 1 #fff);
}

QToolTip {
   border: 2px solid darkkhaki;
   padding: 2px;
   border-radius: 3px;
   opacity: 500;
   font-size: 10px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QToolButton" name="newMolecule">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>New Molecule</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/NewDocument.png</normaloff>:/resources/icons/NewDocument.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="open">
     <property name="toolTip">
      <string>Open File</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/Open.png</normaloff>:/resources/icons/Open.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="save">
     <property name="toolTip">
      <string>Save</string>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/Save.png</normaloff>:/resources/icons/Save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="manipulateMode">
     <property name="toolTip">
      <string>Manipulate Mode</string>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/Manipulate.png</normaloff>:/resources/icons/Manipulate.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buildMode">
     <property name="toolTip">
      <string>Build Mode (Alt)</string>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/Equipment.png</normaloff>:/resources/icons/Equipment.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="elementSelect">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>34</horstretch>
       <verstretch>34</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>38</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Build Element&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;

QToolTip {
   border: 2px solid darkkhaki;
   padding: 2px;
   border-radius: 3px;
   opacity: 200;
   font-size: 10px;
}</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="addFragment">
     <property name="toolTip">
      <string>Add Fragment</string>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/AddFragment.png</normaloff>:/resources/icons/AddFragment.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="addHydrogens">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Add Hydrogens</string>
     </property>
     <property name="text">
      <string>H</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/AddHydrogens.png</normaloff>:/resources/icons/AddHydrogens.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="minimizeEnergy">
     <property name="toolTip">
      <string>Minimize Energy</string>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/Minimize.png</normaloff>:/resources/icons/Minimize.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="selectMode">
     <property name="toolTip">
      <string>Select Mode (Shift)</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/MagicWand.png</normaloff>:/resources/icons/MagicWand.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="deleteSelection">
     <property name="toolTip">
      <string>Delete Selection</string>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/Delete.png</normaloff>:/resources/icons/Delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="takeSnapshot">
     <property name="toolTip">
      <string>Save picture</string>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/Camera.png</normaloff>:/resources/icons/Camera.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="recordAnimation">
     <property name="toolTip">
      <string>Record animation</string>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/Movie.png</normaloff>:/resources/icons/Movie.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="fullScreen">
     <property name="toolTip">
      <string>Full Screen Mode</string>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/Display.png</normaloff>:/resources/icons/Display.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="showHelp">
     <property name="toolTip">
      <string>Show Help</string>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IQmol.qrc">
       <normaloff>:/resources/icons/Help.png</normaloff>:/resources/icons/Help.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>43</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="IQmol.qrc"/>
 </resources>
 <connections/>
</ui>
